sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -687,9 +687,6 @@ export class StudentService extends RecordDataModelService<Student> {
     if (pdVerified) {
       return DisabilityStatus.PD;
     }
-    if (pdVerified === false) {
-      return DisabilityStatus.Declined;
-    }
```

Comment:
Shouldn't we return `DisabilityStatus.Requested` ?

Reply:
That is what we explained in today's meeting. ATBC is not sending any false status. So the default is NotRequested.

Reply:
This transformation is applied on sfas data which comes through the feed. 

As per the discussion at standup today, the user story had wrong information about mapping sfas import pd_status = false -> `DisabilityStatus.Requested`

only mapping what we need to do here is pd_status = false -> `DisabilityStatus.PD`

@guru-aot as discussed we can add the context as comment to the method and extend the comment when we have ppd status